services/event/service/profile_service.go

```diff
@@ -53,3 +53,19 @@ func AwardPoints(id string, points int) (*models.Profile, error) {
 
 	return &profile, nil
 }
+
+func GetIsUserVirtual(id string) (bool, error) {
+	var profile models.Profile
+
+	status, err := apirequest.Get(config.PROFILE_SERVICE+"/profile/user/"+id+"/", &profile)
```

Comment:
you may be able to function call `GetProfile` directly here instead of making a GET request to ourselves

Reply:
`GetProfile` is not part of the Event service, so my thought was we cannot (and probably should not) make calls to functions across services since these are separate processes with separate databases. Hence, why I used a request.

There are some other instances of service methods within the code base where requests are used to retrieve information across services.

Reply:
Yep you're right, I was looking in the wrong service